services/controllerhost/api_handlers.go

```diff
@@ -41,12 +41,12 @@ const (
 )
 
 const (
-	minOpenExtentsForDstPlain                    = 4
-	maxExtentsToConsumeForDstPlain               = 8
-	extentsToConsumePerRemoteZone                = 4
 	minOpenExtentsForDstDLQ                      = 1
 	maxExtentsToConsumeForDstDLQ                 = 2
 	minOpenExtentsForDstTimer                    = 2
+	defaultMinOpenExtents                        = 2 // Only used if the extent configuration can't be retrieved
+	defaultRemoteExtents                         = 2
```

Comment:
naming nit: Can you make this const name specific ? its not clear from the name, if this is a number or a list of extents. Also, is this for publish/consume or both ?